scripts/config.py

```diff
@@ -142,3 +145,7 @@ def set_pinecone_region(self, value: str):
     def set_debug_mode(self, value: bool):
         """Set the debug mode value."""
         self.debug_mode = value
+
+    def set_dns_server(self, value: str):
+        """Set a specific DNS server to use"""
+        self.dns_server = value
```

Comment:
JFYI you need to leave the final line feed as per pep8. But I approve like this.